src/temporal/t.rast.climatologies/t.rast.climatologies.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+
+################################################
+#
+# MODULE:       t.rast.climatologies
+# AUTHOR(S):    Luca Delucchi, Fondazione Edmund Mach
+# PURPOSE:      t.rast.climatologies calculate climatologies for space time raster dataset
+#
+# COPYRIGHT:        (C) 2023 by Luca Delucchi
+#
+#                This program is free software under the GNU General Public
+#                License (>=v2). Read the file COPYING that comes with GRASS
+#                for details.
+#
+################################################
+
+# %module
+# % description: Calculate climatologies in a space time raster dataset
+# % keyword: temporal
+# % keyword: raster
+# % keyword: aggregation
+# % keyword: series
+# % keyword: time
+# %end
+
+# %option G_OPT_STRDS_INPUT
+# % key: strds
+# %end
+
+# %option G_OPT_STRDS_OUTPUT
+# % required: no
+# %end
+
+# %option
+# % key: basename
+# % type: string
+# % label: Basename of the new generated output maps
+# % description: Either a numerical suffix or the start time (s-flag) separated by an underscore will be attached to create a unique identifier
```

Comment:
Why not using granularity as suffix like in other modules?

Reply:
Because I don't want  to mix granularty meaning with something little different